docker2fl/src/docker2fl.rs

```diff
@@ -202,35 +244,41 @@ async fn container_boot(
     let mut env: HashMap<String, String> = HashMap::new();
     let mut cwd = String::from("/");
 
-    let cmd = container_config.cmd.unwrap();
+    let cmd = container_config.cmd.expect("failed to get cmd configs");
 
     if container_config.entrypoint.is_some() {
-        let entrypoint = container_config.entrypoint.unwrap();
-        command = (entrypoint.first().unwrap()).to_string();
+        let entrypoint = container_config
+            .entrypoint
+            .expect("failed to get entry point");
```

Comment:
Why this should cause the entire process to panic? What if the container I am using is not configured with an entrypoint.

I really think all `expect` and `unwrap` here should instead return a suitable errors that can be handled instead of panicing